src/fides/api/models/privacy_experience.py

```diff
@@ -227,10 +228,28 @@ class PrivacyExperienceConfig(
         order_by="ExperienceTranslation.created_at",
     )
 
+    @property
+    def all_regions(self) -> List[PrivacyNoticeRegion]:
+        """All regions physically linked to the PrivacyExperienceConfig regardless of which locations
+        have been configured"""
+        db = Session.object_session(self)
+        if not db:
+            # For dry update scenarios, where PrivacyExperienceConfig is not bound to a session
+            return []
+
+        return [
+            exp.region for exp in self.experiences.order_by(PrivacyExperience.region)
+        ]
+
     @property
     def regions(self) -> List[PrivacyNoticeRegion]:
-        """Return the regions using this experience config"""
-        return [exp.region for exp in self.experiences]  # type: ignore[attr-defined]
+        """Filters regions on configured location if applicable"""
+        db = Session.object_session(self)
+        if not db:
+            # For dry update scenarios
+            return []
```

Comment:
We create a PrivacyExperienceConfig in memory on the dry_update but don't add to the Session.  Now that I'm accessing the object's session as part of the PrivacyExperienceConfig.regions, this would break